ServiceWebsite/ServiceWebsite.AcceptanceTests/TestClients/IApiClient.cs

```diff
@@ -0,0 +1,13 @@
+﻿using RestSharp;
+
+namespace ServiceWebsite.AcceptanceTests.TestClients
+{
+    public interface IApiClient
+    {
+        RestClient Client { get; }
```

Comment:
Should be IRestClient, always use the interfaces

Reply:
removed